holonote/app/panel.py

```diff
@@ -224,57 +224,69 @@ def _callback_apply(self, event):
         elif self._widget_mode_group.value == "-" and selected_ind is not None:
             self.annotator.delete_annotation(selected_ind)
 
-    def _get_layout(self, name):
+    def _add_layout(self, name):
+        """
+        Add a layout to the panel, by cloning the root layout, linking the close button,
+        and returning it visibly.
+        """
+
         def close_layout(event):
             layout.visible = False
 
         layout = self._layouts.get(name)
         if not layout:
-            layout = self._layout.clone(visible=False)
+            layout = self._layout.clone(visible=True)
             self._widget_apply_button.on_click(close_layout)
             self._layouts[name] = layout
         return layout
 
-    def _hide_layouts(self):
-        for layout in self._layouts.values():
-            layout.visible = False
+    def _hide_layouts_except(self, desired_name) -> pn.Column:
+        """
+        Prevents multiple layouts from being visible at the same time.
+        """
+        desired_layout = None
+        with param.parameterized.batch_call_watchers(self):
```

Comment:
Does this do anything? I haven't looked at the code, but it seems like it will only send a batch for the `self` and not underlying widgets/objects. 

Is it to handle `depends` / `watch` on self? Maybe add a comment if its the case.

Reply:
Yep, `batch_call_watchers` does nothing if the updates are spread across multiple components.